docs/aws/aws.md

```diff
@@ -196,3 +196,24 @@ See the egress golang examples above, and replace the line starting with `out :=
 ```go
 out := cli.VerifyDns(context.TODO(), "vpcID")
 ```
+
+## Interface ##
+
+### Platform ###
+The platform interface is used to inform network-verifier of the platform type it is running on. It is used to represent specific Platform types and how they map to their respective platform. It contains an array of length containing valid names for each platform type.
+```
```

Comment:
One suggestion on wording here

> "The `Platform` struct type is used to inform network-verifier of the type of OSD platform it is running on (AWSClassic, GCPClassic, etc). This helps map the various aliases we support to identify a singular platform, for example AWSClassic can be referenced by the aliases `aws-classic` and `aws`.

I am still learning all these components but this is how I am led to understand it, so if I mispoke feel free to let me know and adjust 👍